ultralytics/solutions/serve.py

```diff
@@ -0,0 +1,29 @@
+import base64
+import io
+
+import litserve as ls
+import numpy as np
+from PIL import Image
+
+from ultralytics import YOLO
+
+
+class YOLOServe(ls.LitAPI):
+    def setup(self, device):
+        self.model = YOLO("yolo11n.pt")
```

Comment:
This way, model name is not hardcoded:

```suggestion
class YOLOServe(ls.LitAPI):
    def __init__(self, model_name_or_path="yolo11n.pt"):
        self.model_name_or_path = model_name_or_path
        super().__init__()

    def setup(self, device):
        self.model = YOLO(self.model_name_or_path)
```